source/heroes/ScreensaverHero.hx

```diff
@@ -0,0 +1,47 @@
+package heroes;
+
+import flixel.FlxSprite;
+import flixel.util.FlxColor;
+import flixel.FlxG;
+
+class ScreensaverHero extends FlxSprite {
+    var DirectionX:Float;
+    var DirectionY:Float;
+
+    public function new(X:Float=0, Y:Float=0)
+    {
+        super(X, Y);
+        makeGraphic(20, 40, 0xFFFFFFFF);
+        this.DirectionX = FlxG.random.float(0,1);
+        this.DirectionY = FlxG.random.float(0,1);
+    }
+    override public function update(elapsed:Float)
+    {
+        super.update(elapsed);
+        if(this.DirectionX < .5){
+            this.x += 1;
+        }
+        else{
+            this.x -= 1;
+        }
+        if(this.DirectionY < .5){
+            this.y += 1;
+        }
+        else{
+            this.y -= 1;
+        }
```

Comment:
I think the logic here is a little counter-intuitive, where a smaller randomized direction results in a positive direction of movement.

The ending functionality is fine, but it definitely seems like the kind of thing we'll look back on get a bit confused by.